src/python-common/ceph/tests/test_drive_group.py

```diff
@@ -27,7 +35,114 @@ def test_drivegroup_pattern():
 def test_drive_selection():
     devs = DeviceSelection(paths=['/dev/sda'])
     spec = DriveGroupSpec('node_name', data_devices=devs)
-    assert spec.data_devices.paths == ['/dev/sda']
+    assert all([isinstance(x, Device) for x in spec.data_devices.paths])
+    assert spec.data_devices.paths[0].path == '/dev/sda'
 
     with pytest.raises(DriveGroupValidationError, match='exclusive'):
         DeviceSelection(paths=['/dev/sda'], rotational=False)
+
+
+def test_ceph_volume_command_0():
```

Comment:
these tests really look basically the same as `ceph.tests.test_disk_selector.TestDriveSelection`. As `ceph.tests.test_disk_selector.TestDriveSelection` only tests a subset, we should remove it.